### ISSUE TITLE
MC.2+S:D111N [61 seqs, >10 places,( 16 with S:G261D and 11 with S:Q1201K)]

### ISSUE THREAD

**Issue by xz-keg** [sars-cov-2-variants/lineage-proposals#2107]
S:D111N seems very common among KP.3.1.1 sub-lineages now.
MC.2+G21893A(S:D111N)

GISAID query: G21893A,C12616T,G24095T, -22272
No. of seqs: 10(USA 8(NJ,DE,PA) Canada 1(BC)) 
GISAID query for S:Q1201K branch: G21893A,C25162A,C25163A
Gisaid query for S:G261D branch: Query: G22344A,G21893A,C12616T

First: EPI_ISL_19347326, USA-NJ, 2024-7-31
Latest: EPI_ISL_19445605, USA-NJ, 2024-9-5

[usher](https://nextstrain.org/fetch/genome-test.gi.ucsc.edu/trash/ct/subtreeAuspice2_genome_test_1694e_a299d0.json?label=id:node_5672585)
![image](https://github.com/user-attachments/assets/89901206-2dcc-4e9f-a18a-5af5c9e3ae73)

**Comment by FedeGueli** [comment]
Yeah S:D111N very common and doesnt seem to add anything to Kp.3.1.1 ( a bit as 346T that is popping up everywhere without an advantage)

**Comment by FedeGueli** [comment]
> > Yeah S:D111N very common and doesnt seem to add anything to Kp.3.1.1 ( a bit as 346T that is popping up everywhere without an advantage) Here likely better because of 845 that is the fastest big Kp.3.1.1 branch along with the 572I one
> 
> It also has the double nuc Q1201K in KP.2.1

Yeah it was in  some proposed/designated i doubt it is advantageous at this point

**Comment by FedeGueli** [comment]
11 Denmark, 5 for 1201K

**Comment by FedeGueli** [comment]
13, Ohio

**Comment by xz-keg** [comment]
17, Canada-QC and NS

**Comment by FedeGueli** [comment]
26/6 now

**Comment by xz-keg** [comment]
28, Norway

**Comment by FedeGueli** [comment]
SLowed down

**Comment by xz-keg** [comment]
34, Canada-AB, USA-NH,IA,NY

**Comment by xz-keg** [comment]
+1 UK

**Comment by FedeGueli** [comment]
39 now with a cluster from Quebec with S:261D (G22344A)
Query: G22344A,G21893A,C12616T
Samples: 8: Quebec, Iowa

**Comment by FedeGueli** [comment]
12 for the 261D branch of this reopening it

**Comment by xz-keg** [comment]
61 seqs now
